packages/odyssey-storybook/src/components/odyssey-mui/Accordion/Accordion.stories.tsx

```diff
@@ -123,3 +127,57 @@ export const Multi: StoryObj<AccordionProps> = {
     );
   },
 };
+
+export const Disabled: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+    isDisabled: true,
+  },
+  render: function C(props: AccordionProps) {
+    return (
+      <Accordion label="Title" isDisabled={props.isDisabled}>
+        {props.children}
+      </Accordion>
+    );
+  },
+};
+
+export const Expanded: StoryObj<AccordionProps> = {
+  args: {
+    children: "This is the content of the box.",
+  },
+  render: function C(props: AccordionProps) {
+    const [expanded, setExpanded] = useState(true);
+    const onChange = useCallback(
+      (_event: SyntheticEvent<Element>, expanded: boolean) =>
+        setExpanded(expanded),
+      [],
+    );
```

Comment:
Instead of defining props in the function itself, we should utilize the generics of `useCallback`:

useCallback<AccordionProps["onChange"]>(
      (event:, expanded) =>
        setExpanded(expanded),
      [],
    )